source/zh-tw/docs/github-pages.md

```diff
@@ -30,7 +30,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          # If you have other token like `ssh-key`, please check the usage of this action: https://github.com/actions/checkout#usage
+          token: ${{ secrets.DEPLOY_TOKEN }}
```

Comment:
The `GITHUB_TOKEN` is a special token that is created automatically for each workflow run.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret